pkgs/by-name/lh/lha/package.nix

```
@@ -7,27 +7,30 @@
 
 stdenv.mkDerivation {
   pname = "lha";
-  version = "unstable-2021-01-07";
+  version = "unstable-2024-11-27";
```

```suggestion
  version = "1.14i-unstable-2024-11-27";
```
We would like to add a base version, `1.14i`.

---

Thank you, I fixed it.